src/org/sosy_lab/java_smt/solvers/z3/Z3FormulaCreator.java

```diff
@@ -931,8 +969,17 @@ public void forceClose() {
 
     // Force clean all ASTs, even those which were not GC'd yet.
     // Is a no-op if phantom reference handling is not enabled.
```

Comment:
```suggestion
```
The second line of the context is now redundant because of the explicit condition.